nf_core/pipelines/schema.py

```diff
@@ -869,23 +863,38 @@ def launch_web_builder(self):
         """
         Send pipeline schema to web builder and wait for response
         """
+        # check if self.schema_filename doesn't have uncommited changes using git python
+        if self.schema_filename and self.pipeline_dir:
+            repo = git.Repo(self.pipeline_dir)
+            if str(Path(self.schema_filename).relative_to(self.pipeline_dir)) in [
+                item.a_path for item in repo.index.diff(None)
+            ]:
+                if not Confirm.ask(
+                    f""":exclamation_mark:  '{str(self.schema_filename)}' has uncommitted changes. These will be overwritten in the following steps. Do you still want to continue""",
```

Comment:
I think this is always thrown? I just tested with a fresh pipeline and I see this logging:
```
INFO     [✓] Default parameters match schema validation                                                                       
INFO     [✓] Pipeline schema looks valid (found 32 params)                                                                    
INFO     Writing schema with 32 params: 'nextflow_schema.json'                                                                
🚀  Launch web builder for customisation and editing? [y/n] (y): 
❗  'nextflow_schema.json' has uncommitted changes. These will be overwritten in the following steps. Do you still want to 
continue [y/n] (y): 
```